app/services/taxonomy/publish_taxon.rb

```diff
@@ -19,7 +19,6 @@ def publish(validate: true)
       end
 
       Services.publishing_api.put_content(content_id, payload)
-      Services.publishing_api.publish(content_id, "minor")
```

Comment:
By removing this line, are we still publishing the content item here, instead of just updating it? (Just wasn't sure if it's still fine to have the class name of `PublishTaxon` still or if it now needs to be changed.)